Model/Payment_Method.php

```diff
@@ -166,4 +166,30 @@ public function getConfigFields()
             ],
         ];
     }
+
+    /**
+     * Returns array of setting names which should be part of output debug infos
+     *
+     * @return array
+     */
+    public function getPublicFieldNames()
+    {
+        return ['apiUrl', 'maid', 'descriptor', 'additionalInfo', 'paymentAction'];
+    }
+
+    /**
+     * Returns an array of fields to be displayed in support email
+     *
+     * @return array
+     *
+     * @SuppressWarnings(PHPMD)
```

Comment:
This is not needed.

Reply:
MD check is complaining about unused $field local variable in the anonymous function.